cfgov/v1/wagtail_hooks.py

```diff
@@ -128,46 +129,21 @@ def configure_page_revision(page, is_sharing, is_live):
 
 
 def get_akamai_credentials():
```

Comment:
Previously, `flush_akamai` lived here, so I just added it here too, but now it makes less sense.  I'm not going to worry about it too much for now; I anticipate this code will get shuffled around a bit, particularly if we look into extending Wagtail's frontendcache module with an Akamai backend as @chosak suggested.